templates/BenchmarkDotNet.Templates.csproj

```diff
@@ -24,4 +24,18 @@
     <Compile Remove="**\*" />
     <None Remove="*.bat;.sh" />
   </ItemGroup>
+
+  <Target Name="Fix BenchmarkDotNetVersion"
+          BeforeTargets="Build">
+    <ReplaceFileText
+            InputFilename="templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json"
+            OutputFilename="templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json"
+            MatchExpression='("defaultValue": )"([\d+\.?]+)"'
+            ReplacementText='${1}"$(Major).$(Minor).$(Revision)"' />
+    <PropertyGroup>
```

Comment:
It seems that we don't actually need this PropertyGroup: all the arguments are defined in attributes of `ReplaceFileText`.